### PR TITLE
Jamelle wt applying loan 1

### PR DESCRIPTION
**Pull request by JamelleWT**

### CODE REVIEW

**Comment by JamelleWT** [comment]
@KishaT ready for your review

File: pages/loan-basics/applying-for-loan.md [hunk 1]
```diff
@@ -12,3 +12,64 @@ scripts:
...
+</div>
+ 
+## Spousal rules
+By law, your spouse has certain rights to your TSP account. When requesting a loan, you must indicate if you are married, even if you are separated from your spouse.
++ FERS or uniformed services participants: Your spouse must consent to your TSP loan by signing the Loan Agreement that you print from this website.
++ CSRS participants: We must notify your spouse when you apply for a loan.
+Exceptions to the spousal rule may be approved under very limited circumstances. For more information, please refer to [Form TSP-16](#), _Exception to Spousal Requirements_ ( [TSP-U-16](#), for uniformed services members), which is available from your agency or service.
+## Residential loan documentation
+When you request a residential loan, you must complete the Residential Loan Agreement and submit it, along with any required documentation, before your loan can be approved. This form documents the costs associated with the purchase or construction of your primary residence.
+#### Documentation requirements
```
Comment (line47) [inline pages/loan-basics/applying-for-loan.md]: This should be a level 3 header `###`

File: pages/loan-basics/applying-for-loan.md [hunk 2]
```diff
@@ -12,3 +12,64 @@ scripts:
...
+_**All residential loans**_
+All of your documentation must
++ be from a third party (that is, someone who is selling the home to you or building it for you); and
++ be dated no more than 24 months before the expiration date on the TSP Loan Agreement.
+Your documentation must also show
++ you or your spouse as the purchaser or builder;
++ the purchase or construction price; and
++ the full address of the primary residence.
+##### Special requirements
```
Comment (line47) [inline pages/loan-basics/applying-for-loan.md]: This should be a level 4 header `####`

File: pages/loan-basics/applying-for-loan.md [hunk 3]
```diff
@@ -12,3 +12,64 @@ scripts:
...
+In addition to the requirements listed above, some types of primary residences require you to show further documentation.
+##### Cooperative purchase
```
Comment (line47) [inline pages/loan-basics/applying-for-loan.md]: This should be a level 4 header `####`

File: pages/loan-basics/applying-for-loan.md [hunk 4]
```diff
@@ -12,3 +12,64 @@ scripts:
...
+Purchasing a cooperative requires you to show that you own shares in a cooperative housing corporation.
+##### New home construction purchase
```
Comment (line47) [inline pages/loan-basics/applying-for-loan.md]: This should be a level 4 header `####`

File: pages/loan-basics/applying-for-loan.md [hunk 5]
```diff
@@ -12,3 +12,64 @@ scripts:
...
+Constructing a new primary residence requires you to show evidence that a complete residence will be constructed, such as:
++ Building permits
++ Building receipts
++ Other third-party documents
+##### Mobile home or recreational vehicle purchase
```
Comment (line47) [inline pages/loan-basics/applying-for-loan.md]: This should be a level 4 header `####`

File: pages/loan-basics/applying-for-loan.md [hunk 6]
```diff
@@ -12,3 +12,64 @@ scripts:
...
+Purchasing a mobile home or recreational vehicle as a primary residence requires you to provide all of the following:
++ Serial number or other identifying number
++ Complete description of facilities and accommodations (RVs)
++ Signed statement that the RV will be your primary residence (RVs)
+##### Boat purchase
```
Comment (line47) [inline pages/loan-basics/applying-for-loan.md]: This should be a level 4 header `####`

File: pages/loan-basics/benefits-and-limitations.md [hunk 7]
```diff
@@ -12,3 +12,100 @@ scripts:
...
+## How a TSP loan works
+When you take a loan, you borrow from your TSP account. The amount of your loan cannot exceed the amount of your own contributions and earnings from those contributions. So, if you receive contributions from your agency or service, you cannot borrow from those contributions or earnings from those contributions.
+ 
+If you meet the loan eligibility rules and your loan request is approved, the loan amount is removed from your TSP account. You must repay your loan with interest. Repayments for the loan and interest are generally made through payroll deductions. Your repayments restore the amount of your loan, plus interest, to your account.
+For additional information, visit [Repaying your loan](repaying-your-loan.md).
+
+## Loan types and terms
+We allow two types of loans:
+
+#### General purpose
```
Comment (line47) [inline pages/loan-basics/benefits-and-limitations.md]: This should be a level 3 header `###`

File: pages/loan-basics/benefits-and-limitations.md [hunk 8]
```diff
@@ -12,3 +12,100 @@ scripts:
...
++ May be used for any purpose
++ Requires no documentation
++ Has a repayment term of 1 to 5 years
+
+#### Residential
```
Comment (line47) [inline pages/loan-basics/benefits-and-limitations.md]: This should be a level 3 header `###`

File: pages/loan-basics/benefits-and-limitations.md [hunk 9]
```diff
@@ -12,3 +12,100 @@ scripts:
...
+<div class="usa-alert-body" markdown="1">
+
+Your loan payments must start within 60 days of your loan being disbursed.
+{: .usa-alert-text}
+</div>
+</div>
+
+## Loan eligibility
+
+#### General eligibility rules
```
Comment (line47) [inline pages/loan-basics/benefits-and-limitations.md]: This should be a level 3 header `###`

File: pages/loan-basics/benefits-and-limitations.md [hunk 10]
```diff
@@ -12,3 +12,100 @@ scripts:
...
+**To be eligible for a loan, you**
++ must be an active federal employee or a member of the uniformed services.
++ must be in pay status because repayments are set up as payroll deductions.
++ can only have one outstanding general purpose loan and one outstanding residential loan from any one TSP account at a time.
++ must have at least $1,000 of your own contributions and earnings in your account (agency contributions and earnings cannot be borrowed).
++ must not have repaid a loan of the same type in full within the past 60 days. (If you have both a civilian account and a uniformed services account, the 60-day waiting period applies separately to each account.)
++ must not have had a taxable distribution of a loan within the past 12 months unless it was the result of your separation from federal service.
++ must not have a court order against your account.
+
+#### Residential loan eligibility rules
```
Comment (line47) [inline pages/loan-basics/benefits-and-limitations.md]: This should be a level 3 header `###`

File: pages/loan-basics/benefits-and-limitations.md [hunk 11]
```diff
@@ -12,3 +12,100 @@ scripts:
...
+. . + refinancing or prepaying your existing mortgage;
+. . + construction of an addition to your existing residence;
+. . + renovations to your existing residence;
+. . + buying out another person's share in the your current residence; or
+. . + the purchase of land only.
+ 
++ Your primary residence must be purchased in whole or in part by you, or your spouse.
+
+## Borrowing limits
+#### Minimum loan amount
```
Comment (line47) [inline pages/loan-basics/benefits-and-limitations.md]: This should be a level 3 header `###`

File: pages/loan-basics/benefits-and-limitations.md [hunk 12]
```diff
@@ -12,3 +12,100 @@ scripts:
...
+The minimum loan amount that you can borrow is $1,000. The amount of your account balance that consists of your own contributions and earnings on those contributions, must be at least equal to the minimum loan amount. 
+#### Maximum loan amount
```
Comment (line47) [inline pages/loan-basics/benefits-and-limitations.md]: This should be a level 3 header `###`

File: pages/loan-basics/benefits-and-limitations.md [hunk 13]
```diff
@@ -12,3 +12,100 @@ scripts:
...
+<div class="usa-alert usa-alert-info">
+<div class="usa-alert-body" markdown="1">
+
+To find out more about the maximum amount available for a loan, use the calculator, [ _Estimate Loan Payments_](#) , or contact us.
+{: .usa-alert-text}
+</div></div>
+
+## Loan costs
+
+#### Direct costs
```
Comment (line47) [inline pages/loan-basics/benefits-and-limitations.md]: This should be a level 3 header `###`

File: pages/loan-basics/benefits-and-limitations.md [hunk 14]
```diff
@@ -12,3 +12,100 @@ scripts:
...
+
+_**Loan Fee**_ We charge a loan fee of $50 for administrative expenses. This fee is deducted from your loan proceeds. For example, if you request a loan for $1,000, the amount paid to you will be $950.
+ 
+_**Interest**_ The interest rate on your loan is the G Fund rate at the time your loan application is processed. This rate is fixed for the life of the loan. Although loan interest is not tax-deductible, all of the interest goes back into your TSP account.
+#### Indirect costs
```
Comment (line47) [inline pages/loan-basics/benefits-and-limitations.md]: This should be a level 3 header `###`

File: pages/loan-basics/applying-for-loan.md (hunk 1 above)
Comment (KishaT) [reply pages/loan-basics/applying-for-loan.md]: @line47 Yes, it should. Please make the change(s) for the appropriate headers as necessary. Thank you.

**Comment by KishaT** [comment]
@line47 Please make the changes to all the headers as necessary.

**Comment by line47** [comment]
@KishaT I'm happy to make the change but would prefer for @JamelleWT to make the updates if possible (let me know if you have any questions)

**Comment by KishaT** [comment]
@line47 OK, sounds good. @JamelleWT  is off today, so it'll have to wait until Monday. Thanks Scott.

**Pull request closed without merging**